src/main/java/com/example/AkkaMainSystem.java

```diff
@@ -24,9 +36,15 @@ public Receive<Create> createReceive() {
 
     private Behavior<Create> onCreate(Create command) {
         //#create-actors
-        ActorRef<Library.Message> library = this.getContext().spawn(Library.create("library"), "library");
-        ActorRef<QueueManager.Message> queueManager = this.getContext().spawn(QueueManager.create("QueueManager"), "queueManager");
-        ActorRef<Spawner.Message> spawner = this.getContext().spawn(Spawner.create("Spawner"), "spawner");
+        songList.add(songD1);
+        songList.add(songD2);
+        songList.add(songD3);
+        songList.add(songBB1);
+        songList.add(songES1);
+        songList.add(songTS1);
```

Comment:
das macht vlt nicht viel aus, aber die Künstler Initialisierung und Hinzufügen von Lieder kann man in eine Methode packen, ist etwas cleaner

Reply:
stimmt. Das können wir ja machen